releaseNotes/Microsoft.Featuremanagement.md

```diff
@@ -2,6 +2,17 @@
 
 [Source code][source_code] | [Package (NuGet)][package] | [Samples][samples] | [Product documentation][docs]
 
+## 3.3.0 - May 8, 2024
+
+### Enhancements
+
+* Added a `Recurrence` option to the `TimeWindow` filter to support recurring time windows. This enables scenarios such as enabling the flag every day after five PM or enabling the flag on Fridays. See more details [here](https://github.com/microsoft/FeatureManagement-Dotnet?tab=readme-ov-file#recurrence-pattern). ([#266](https://github.com/microsoft/FeatureManagement-Dotnet/pull/266))
```

Comment:
Added a `Recurrence` option to the `TimeWindow` filter to support recurring time window. This enables scenarios where a feature flag is activated based on recurrence patterns, such as every day after 5 PM or every Friday.
```suggestion
* Added a `Recurrence` option to the `TimeWindow` filter to support recurring time window. This enables scenarios where a feature flag is activated based on recurrence patterns, such as every day after 5 PM or every Friday. See more details [here](https://github.com/microsoft/FeatureManagement-Dotnet?tab=readme-ov-file#recurrence-pattern). ([#266](https://github.com/microsoft/FeatureManagement-Dotnet/pull/266))
```